sys/kern/vfs_lookup.c

```diff
@@ -842,34 +842,35 @@ vfs_lookup_failifexists(struct nameidata *ndp)
  *
  * The pathname is pointed to by cn_nameptr and is of length ni_pathlen.
  * The starting directory is taken from ni_startdir. The pathname is
- * descended until done, or a symbolic link is encountered. The variable
- * ni_more is clear if the path is completed; it is set to one if a
+ * descended until done, or a symbolic link is encountered. The cn_flags
+ * has ISLASTCN or'ed if the path is completed or ISSYMLINK or'ed if a
  * symbolic link needing interpretation is encountered.
```

Comment:
It looks that some time ago it was passed as ni_more variable as a flag, today it works with cn_flags bits instead.